From 13210fca9a8a95f89d15dd8dea76460c7b1fdb38 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Sat, 15 Sep 2018 11:41:05 +0900
Subject: [PATCH 563/909] arm64: dts: r8a77990-ebisu-4d: Add lossy descompress
 area

commit 1926d72f2029b09988999329f8fcbcc346c2f55a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds the memory reserved for the lossy compression.

The added reserved memory region is as follows:
  - Lossy Decompression
       48 MiB : 0x0054000000 -> 0x0056ffffff

[takeshi.kihara.df: fix W=1 dtc unit_address_vs_reg warnings]
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts
index 0e7ab038fd43..49017ce0d800 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu-4d.dts
@@ -17,6 +17,14 @@
 		/* first 128MB is reserved for secure area. */
 		reg = <0x0 0x48000000 0x0 0x78000000>;
 	};
+
+	reserved-memory {
+		/* device specific region for Lossy Decompression */
+		lossy_decompress: linux,lossy_decompress@54000000 {
+			no-map;
+			reg = <0x00000000 0x54000000 0x0 0x03000000>;
+		};
+	};
 };
 
 &pciec0 {
-- 
2.17.1

